rest_framework_serializer_field_permissions/fields.py

```diff
@@ -76,8 +76,11 @@ class BooleanField(PermissionMixin, fields.BooleanField):
     pass
 
 
-# pylint: disable=missing-docstring
-class NullBooleanField(PermissionMixin, fields.NullBooleanField):
+try:
+    # pylint: disable=missing-docstring
+    class NullBooleanField(PermissionMixin, fields.NullBooleanField):
+        pass
+except AttributeError:
     pass
```

Comment:
@gquittet I've created this alternate pull request which I believe should address the error that you've encountered. Rather than removing the `NullBooleanField` entirely, I instead _try_ to make a `NullBooleanField` but if I encounter an error then I swallow it and move on.

I've released this change as version 4.1.0rc1 of the library, which you can see at https://pypi.org/project/django-rest-serializer-field-permissions/#history . Could you try this version in your application and confirm that it works for you? After that, I'll make a full release of 4.1.0.